bogota-abril-2019/css-magazine-cover/david-villamizar/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8" />
+    <title>WIRED UK Amazon's Next Move</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link rel="manifest" href="site.webmanifest" />
+    <link rel="apple-touch-icon" href="icon.png" />
+    <!-- Place favicon.ico in the root directory -->
+
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/main.css" />
+
+    <meta name="theme-color" content="#fafafa" />
+  </head>
+
+  <body>
+    <!--[if IE]>
+      <p class="browserupgrade">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="https://browsehappy.com/">upgrade your browser</a> to improve
+        your experience and security.
+      </p>
+    <![endif]-->
+
+    <!-- Add your site or application content here -->
+    <div id="magazine-cover">
+      <img id="fake-jeff" src="img/fake-bezos.jpg" alt="Fake Jeff Bezos" />
+      <h1 id="magazine-title">
+        <img src="img/wired-logo.svg" alt="WIRED Logo" />
+        <span>
+          UK Edition
+        </span>
+      </h1>
+      <h2 id="issue-title">Amazon's Next Move</h2>
+      <section>
+        <h3>Exclusive</h3>
+        <h4>From Alexa to Drones:</h4>
+        <h5>The Jeff Bezos Master Plan</h5>
+      </section>
+      <section>
+        <span>Plus</span>
+        <hr />
+        <h3>Product Special</h3>
+        <h4>WIRED's Ultimate Gear Guide</h4>
```

Comment:
Uppercase unnecessary please check https://css-tricks.com/almanac/properties/t/text-transform/

Reply:
The name of the magazine WIRED is always written in uppercase.
![image](https://user-images.githubusercontent.com/4100566/56774820-ddbf6800-6789-11e9-9515-247863215e2a.png)

Reply:
I made a change to address this, since I read screen readers think uppercase words are meant to be read letter by letter.